ui/components/multichain/multichain-token-list-item/multichain-token-list-item.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import classnames from 'classnames';
+import {
+  BLOCK_SIZES,
+  DISPLAY,
+  FLEX_DIRECTION,
+  FONT_WEIGHT,
+  JustifyContent,
+  Size,
+  TextColor,
+  TextVariant,
+  TEXT_ALIGN,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarNetwork,
+  AvatarToken,
+  BadgeWrapper,
+  Text,
+} from '../../component-library';
+import Box from '../../ui/box/box';
+import { getNativeCurrencyImage } from '../../../selectors';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import Tooltip from '../../ui/tooltip';
+
+export const MultichainTokenListItem = ({
+  className,
+  onClick,
+  tokenSymbol,
+  tokenImage,
+  primary,
+  secondary,
+  title,
+}) => {
+  const primaryTokenImage = useSelector(getNativeCurrencyImage);
+  const t = useI18nContext();
+  const dataTheme = document.documentElement.getAttribute('data-theme');
+  return (
+    <Box
+      className={classnames('multichain-token-list-item', className)}
+      display={DISPLAY.FLEX}
+      flexDirection={FLEX_DIRECTION.COLUMN}
+      gap={4}
+      data-testid="multichain-token-list-item"
+    >
+      <Box
+        className="multichain-token-list-item__container-cell"
+        display={DISPLAY.FLEX}
+        flexDirection={FLEX_DIRECTION.ROW}
+        padding={4}
+        as="a"
+        data-testid="multichain-token-list-button"
+        href="#"
+        onClick={(e) => {
+          e.preventDefault();
+          onClick();
+        }}
+      >
+        <BadgeWrapper
+          badge={
+            <AvatarNetwork
+              size={Size.XS}
+              name={t('networkNameEthereum')}
+              src={primaryTokenImage}
+            />
+          }
+          marginRight={3}
+        >
+          <AvatarToken name={tokenSymbol} src={tokenImage} showHalo />
+        </BadgeWrapper>
```

Comment:
You can add some logic here so that borders show if images aren't available. To really refine the component we will have to make some updates to the `AvatarNetwork` component. I'll create a branch essentially what we will achieve is this


### Before
<img width="279" alt="Screenshot 2023-03-15 at 7 01 01 PM" src="https://user-images.githubusercontent.com/8112138/225490415-be123d1a-3d07-4957-a070-df5de147f12d.png">


### After

<img width="325" alt="Screenshot 2023-03-15 at 6 59 49 PM" src="https://user-images.githubusercontent.com/8112138/225490288-fbfb21ed-16ac-4537-8c6a-c03ba793c1a7.png">


```suggestion
        <BadgeWrapper
          badge={
            <AvatarNetwork
              size={Size.XS}
              name={t('networkNameEthereum')}
              src={primaryTokenImage}
              borderColor={
                primaryTokenImage
                  ? BorderColor.borderMuted
                  : BorderColor.borderDefault
              }
            />
          }
          marginRight={3}
        >
          <AvatarToken
            name={tokenSymbol}
            src={tokenImage}
            showHalo
            borderColor={
              tokenImage ? BorderColor.transparent : BorderColor.borderDefault
            }
          />
        </BadgeWrapper>
```